webapp/src/app/home/home.component.ts

```diff
@@ -33,15 +33,33 @@ export class HomeComponent {
 
   private readonly route = inject(ActivatedRoute);
   private queryParams = toSignal(this.route.queryParamMap, { requireSync: true });
-  protected after = computed(() => this.queryParams().get('after') ?? dayjs().isoWeekday(1).format('YYYY-MM-DD'));
-  protected before = computed(() => this.queryParams().get('before') ?? dayjs().format('YYYY-MM-DD'));
+  protected leaderboardSchedule = computed(() => {
```

Comment:
maybe also lines between computed properties, so it is clearer where each property ends

Reply:
Just added a few more ones to make it more readable. 
I usually prefer a content-based grouping to reduce the time it takes to find correlated code.